# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini
# Commit: d7e962f7ca16
# Full Hash: d7e962f7ca165897bc5c1e32f8a1bcc72ab24629
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2020-01-22 21:48:55
# Description:
#   Bug 1607671. Back out changeset edf3df2adebb (bug 1602410) for regressions. kmag r=RyanVM
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D59785
# ==============================================================================

diff -r 61c5e64eb2b4 -r d7e962f7ca16 testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini
--- a/testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini	Wed Jan 22 14:41:47 2020 +0000
+++ b/testing/web-platform/meta/printing/print-microtask-after-navigate.html.ini	Wed Jan 22 14:43:46 2020 +0000
@@ -1,5 +1,4 @@
 [print-microtask-after-navigate.html]
-  disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1569400#c22
   expected:
     if (os == "android") and not debug: ["ERROR", "OK"]
     if (os == "android") and debug: ERROR